src/test/resources/application-unit-test.yml

```diff
@@ -0,0 +1,4 @@
+workspace:
+  crl:
+    use-crl: false
+    use-janitor: false
```

Comment:
worth a quick comment? `// Unit tests should not be communicating with clouds, so they should not use CRL or Janitor.`